styles.css

```diff
@@ -0,0 +1,81 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0 24px;
+  height: auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 0;
+  margin-top: 58px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+}
+
+.icon {
+  width: 24px;
+  height: 24px;
+  margin-right: 12px;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 100px;
+  padding-top: 10px;
+  align-items: flex-start;
+  background-image: url("./images/header-bg.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+h1 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 60px;
+  margin-top: 100px;
+  vertical-align: 24px;
+}
```

Comment:
- [ ] Kindly change the `font-weight: bold;` to something a litter bigger like `font-weight: 800;` to be like the design.

Reply:
@keroloussamy . Kindly review changes made to the code as was stated.
This is  my last review and I want to make sure before sending it for final review